### PR TITLE
relative positions

### PR DESCRIPTION
**Pull request by nearnshaw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -620,7 +616,7 @@ When instancing a new Option Prompt, you can pass the following parameters:
 ```ts
 ReactEcsRenderer.setUiRenderer(ui.render)
 
-const prompt = ui.createComponent(ui.OptionPrompt, {
+const optionPrompt = ui.createComponent(ui.OptionPrompt, {
```
Comment (AlejandroAlvarezMelucciDCL) [inline README.md:619]: I think we should also rename `prompt` in line 635:
`prompt.show()` to `optionPrompt.show()`

File: src/ui-entities/prompts/Prompt/components/Switch/index.tsx [hunk 2]
```diff
@@ -152,7 +157,8 @@ export class PromptSwitch extends InPromptUIObject {
           flexDirection: 'row',
           alignItems: 'center',
           justifyContent: 'center',
-          positionType: 'absolute',
+          positionType: this.absolute ? 'absolute' : 'relative',
+          margin: { right: 10, left: 10, top: 25, bottom: 25 },
           position: { bottom: this._yPosition, right: this._xPosition * -1 },
         }}
```
Comment (AlejandroAlvarezMelucciDCL) [inline src/ui-entities/prompts/Prompt/components/Switch/index.tsx:163]: Is the double `{{` and `}}` intended?

File: src/ui-entities/prompts/Prompt/components/Icon/index.tsx [hunk 3]
```diff
@@ -77,11 +81,12 @@ export class PromptIcon extends InPromptUIObject {
     this.xPosition = xPosition
     this.yPosition = yPosition
     this.image = image
+    this.absolute = positionAbsolute
     if (section) this.section = section
 
     this.imageElement = {
       uiTransform: {
-        positionType: 'absolute',
+        positionType: this.absolute,
```
Comment (AlejandroAlvarezMelucciDCL) [inline src/ui-entities/prompts/Prompt/components/Icon/index.tsx:89]: You changed `positionType` to use a string elsewhere but here `this.absolute` is a boolean, I think that might cause unexpected behavior, can you check?

File: src/ui-entities/prompts/Prompt/index.tsx [hunk 4]
```diff
@@ -229,9 +240,16 @@ export class Prompt extends UIObject implements IPrompt {
...
+              // textureMode: 'nine-slices',
               texture: {
                 src: this._texture,
               },
+              // textureSlices: {
+              //   top: 0.2,
+              //   bottom: 0.2,
+              //   left: 0.2,
+              //   right: 0.2
+              // },
```
Comment (AlejandroAlvarezMelucciDCL) [inline src/ui-entities/prompts/Prompt/index.tsx:252]: Maybe we can delete commented out code? As well as in line 243

**Review by AlejandroAlvarezMelucciDCL — CHANGES_REQUESTED** [review]
Looks good overall but I'd like you to double check some comments, I might be wrong and no action is needed, let me know and I'll change the approval 😄

File: README.md (hunk 1 above)
Comment (nearnshaw) [reply README.md:619]: Yep, this is from making examples copying and pasting the same code XD

File: src/ui-entities/prompts/Prompt/components/Switch/index.tsx (hunk 2 above)
Comment (nearnshaw) [reply src/ui-entities/prompts/Prompt/components/Switch/index.tsx:163]: It is intended
It's a bit weird if you ask me, but that's how our UIs are written for some reason
https://docs.decentraland.org/creator/development-guide/sdk7/onscreen-ui/

File: src/ui-entities/prompts/Prompt/components/Icon/index.tsx (hunk 3 above)
Comment (nearnshaw) [reply src/ui-entities/prompts/Prompt/components/Icon/index.tsx:89]: thx, good point, this property doesn't know what to do with a boolean
I just replaced it with
this.absolute ? 'absolute' : 'relative',

File: src/ui-entities/prompts/Prompt/index.tsx (hunk 4 above)
Comment (nearnshaw) [reply src/ui-entities/prompts/Prompt/index.tsx:252]: yes, for sure
I was just leaving this in for now while I tried things semi randomly without knowing if I have to uncomment them :p

**Review by AlejandroAlvarezMelucciDCL — APPROVED** [review]
LGTM!

**Pull request merged**